src/graph-helpers/graph-model.js

```diff
@@ -0,0 +1,15 @@
+export default class GraphModel {
+  constructor(attrs) {
+    this.attrs = attrs;
+
+    Object.keys(this.attrs).filter(key => {
+      return !(key in this);
```

Comment:
I'm never sure what the best solution is here but I don't know about silently just dropping these. Could easily lead to problems debugging. What about a message through `console.warn` (think that's non-standard)?